src/data/showcases.js

```diff
@@ -1202,6 +1202,15 @@ export const Showcases = [
     source: null,
     tags: ["governance", "analytics"],
   },
+  {
+    title: "Medusa Wallet",
+    description:
+        "A lightweight Cardano wallet focused on maximum privacy and user protection, enabling easy and secure access to funds even in untrusted or compromised environments.",
```

Comment:
```suggestion
        "A lightweight Cardano wallet focused on privacy and user protection, enabling easy and secure access to funds even in untrusted or compromised environments.",
```
@Fell-x27 I would recommend removing this word because "maximum" privacy, as I can verify from my own work in that area, is something impossible to assure... since you can always increase privacy as long as you don't remind decreasing usability.

Therefore the word "optimal" might be better, but that's a subjective decision: so best of all I think to leave the word out.  "Focused on privacy" will convey to users that it's a priority in your design.

Reply:
Accepted